src/main/java/com/ibm/eventstreams/kafkaconnect/plugins/xml/engines/StructToXmlBytes.java

```diff
@@ -264,30 +262,25 @@ private void processMap(Document doc, Element mapElement, Map<?, ?> map) {
 
             if (key == null) {
                 addNode(doc, mapItemElement, "key");
-            }
-            else if (key instanceof Struct) {
+            } else if (key instanceof Struct) {
                 final Element mapItemKeyElement = addNode(doc, mapItemElement, "key");
                 processStruct(doc, mapItemKeyElement, (Struct) key);
-            }
-            else if (key instanceof String) {
+            } else if (key instanceof String) {
```

Comment:
```suggestion
            } 
            else if (key instanceof String) {
```

Unrelated to this specific PR